src/spaceObjects/spaceObject.cpp

```diff
@@ -304,6 +304,24 @@ void SpaceObject::drawOnGMRadar(sf::RenderTarget& window, sf::Vector2f position,
 {
 }
 
+std::unordered_map<string, string> SpaceObject::getGMInfo()
+{
+    std::unordered_map<string, string> ret = std::unordered_map<string, string>();
+
+    ret["Scanning Depth * Compl."] = string(scanning_depth_value) + " * " + string(scanning_complexity_value);
+
+    if (object_description.not_scanned != "")
+       ret["Descr. 0 (NotSc)"] = object_description.not_scanned;
```

Comment:
Descriptions are be long and are part of the tweak menu, don't think it is smart to put them here.